public/index.html

```diff
@@ -7,18 +7,20 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
   <meta name="description" content="Onboarding. Registra in Developers il tuo strumento di code hosting.">
-  <meta name="author" content="Team per la Trasformazione Digitale">
+  <meta name="author" content="Dipartimento per la Trasformazione Digitale">
   <meta name="robots" content="noindex">
 
   <title>Onboarding · Registra in Developers il tuo strumento di code hosting</title>
 
   <!-- Bootstrap core CSS -->
   <link href="bootstrap-italia/dist/css/bootstrap-italia.min.css?201805161459" rel="stylesheet">
-  <link href="assets/css/main.css?201805161459" rel="stylesheet">
 
   <!-- Documentation extras -->
   <link href="assets/css/docs.min.css?201805161459" rel="stylesheet">
 
+  <!-- Custom CSS -->
+  <link href="assets/css/main.css?201805161459" rel="stylesheet">
```

Comment:
Changed the CSS load pipeline to prioritize our custom changes. What do you think @sebbalex?

Reply:
good catch 👍